spk/lidarr/Makefile

```diff
@@ -0,0 +1,43 @@
+SPK_NAME = lidarr
+SPK_VERS = $(shell date +%Y%m%d)
+SPK_REV = 1
+SPK_ICON = src/lidarr.png
+
+REQUIRED_DSM = 5.0
+UNSUPPORTED_ARCHS = $(PPC_ARCHES)
+
+DEPENDS = cross/curl cross/mediainfo cross/sqlite cross/lidarr
```

Comment:
mediainfo is a really large package... If lidarr invokes it as a fork process, I would recommend a package dependency and configuration of PATH to have commands available to the application
... if it was possible to do the same with curl and sqlite, that package may be non-arch or I am wrong?